### PR TITLE
Update INSTALL.txt

### PR DESCRIPTION
**Pull request by markmckinnon**
update the install.txt file to have information about lvm/libbfio/vhdi/vmdk   Needs to be merged when PR 2751 LVM Support is merged in.

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
     https://github.com/libyal/libewf
     Available at: http://sourceforge.net/projects/libewf/
 
+- LibVHDI: Allows you to process disk images that are stored in the
+Virtual Hard Disk format (VHD).  
+
+    The official repository is available hereL:
```
Comment (joachimmetz) [inline INSTALL.txt:60]: `hereL:` => `here:` (remove the L) (2 more times below)

File: INSTALL.txt [hunk 2]
```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
     https://github.com/libyal/libewf
     Available at: http://sourceforge.net/projects/libewf/
 
+- LibVHDI: Allows you to process disk images that are stored in the
```
Comment (joachimmetz) [inline INSTALL.txt:57]: technically the project name is libvhdi (without the casing)

File: INSTALL.txt [hunk 3]
```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
...
+
+- LibVMDK: Allows you to process disk images that are stored in the
+VMware Virtual Disk format (VMDK).  
+
+    The official repository is available hereL:
+
+    https://github.com/libyal/libvmdk
+
+- Libvslvm: Allows you to access the Linux Logical Volume Manager (LVM) format 
+that is sotred on a disk image.  Libbfio is a requirement for Libvslvm to work:
```
Comment (joachimmetz) [inline INSTALL.txt:72]: `Libbfio is a requirement for Libvslvm to work:` maybe change to: `A stand-alone version of libbfio is needed to allow libvslvm to directly read from a TSK_IMAGE`

File: INSTALL.txt [hunk 4]
```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
     https://github.com/libyal/libewf
```
Comment (joachimmetz) [inline INSTALL.txt:54]: this should be https://github.com/libyal/libewf-legacy

File: INSTALL.txt (hunk 2 above)
Comment (markmckinnon) [reply INSTALL.txt]: Changed.

**Pull request merged**